www/app.py

```diff
@@ -56,6 +103,13 @@ def init_rover_service():
     pwm.start("XIO-P6", 0);
     pwm.start("XIO-P7", 0);
 
+    # set up IR sensor gpio
+    gpio.setup("XIO-P2", gpioLib.IN)
+    gpio.setup("XIO-P4", gpioLib.IN)
+    global binary_sensors
```

Comment:
Can the socketio and flask handler functions be part of classes so that we can construct them in a way to avoid global variables?